tests/testcases/longevity-apps-test/README.md

```diff
@@ -0,0 +1,12 @@
+## longevity-apps-test
+
+longevity test is to ensure `wasm` file(s) compiled with a version of `Spin` continues to work with runtime of future version of `Spin`. 
```

Comment:
Wondering how we anticipate handling failures here while we're pre-1.0?  Perhaps on a case-by-case basis?  (For example, when a breaking change is necessary and/or the preferred option -- I suppose the PR contributor would re-generate the wasm file(s) in question?)

Reply:
from what I gathered from discussion I had with @itowlson, the `wasm` module compiled once should continue to work with future spin runtime. 

but in any case, I will add instructions on how to generate these wasm files.

Reply:
added the instructions to readme.

Reply:
I like the idea of keeping "old" Wasm files and applications around.  I agree that if we were on early 0.x it would be a pain handling failures from known breaking changes.  But now we are close enough to stabilisation that any breaking change from here on would be worth flagging up.  Once we have 1.0 we could biff the pre-1.0 builds and only retain the 1.x cycle; but for now apps built on 0.9 and 0.10 should likely be keep working going into the 1.0 release cycle.